packages/teleport/src/Sessions/SessionList/SessionList.tsx

```diff
@@ -98,3 +111,21 @@ type Props = {
   sessions: Session[];
   pageSize?: number;
 };
+
+function participantMatcher(
+  targetValue: any,
+  searchValue: string,
+  propName: keyof Session & string
+) {
+  if (propName === 'parties') {
+    return targetValue.some((participant: Participant) => {
+      if (participant.remoteAddr.toLocaleUpperCase().includes(searchValue)) {
+        return true;
+      }
+
+      return participant.user.toLocaleUpperCase().includes(searchValue);
+    });
+  }
+  // No match.
+  return false;
```

Comment:
do we really need this? i think it returns undefined if no return statement and that's a falsey right?

Reply:
correct, and the `.some` would actually return `false`